scripts/update_template_files.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script evaluates the entries in .static_files, .mandatory_files and
+.deperecated_files and compares them with the microservice template repository,
+or verifies their existence or non-existence depending on the list they are in.
+"""
+
+import difflib
+import shutil
+import sys
+import urllib.parse
+from pathlib import Path
+
+import requests
+import typer
+from script_utils.cli import echo_failure, echo_success
+
+REPO_ROOT_DIR = Path(__file__).parent.parent.resolve()
+
+DEPRECATED_FILES = ".deprecated_files"
+MANDATORY_FILES = ".mandatory_files"
+STATIC_FILES = ".static_files"
+IGNORE_FILES = ".ignore_files"
```

Comment:
Maybe add one ignored list for deprecated, mandatory, and static_files.

Reply:
Ok, using three different lists with `_ignore` suffix now